File: TransCoder_tokenized_test_set_functions/java/MINIMUM_COST_CONNECT_WEIGHTED_NODES_REPRESENTED_ARRAY.java
static int minimum_cost ( int a [ ], int n ) {
  int mn = Integer . MAX_VALUE;
  int sum = 0;
  for ( int i = 0;
  i < n;
  i ++ ) {
    mn = Math . min ( a [ i ], mn );
    sum += a [ i ];
  }
  return mn * ( sum - mn );
}

File: results/py/MINIMUM_COST_CONNECT_WEIGHTED_NODES_REPRESENTED_ARRAY.cpp
double f_filled ( double * a , int n ) {
  double mn = std :: numeric_limits < double > :: max ( ) ;
  double sum = 0 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    mn = std :: min ( a [ i ] , mn ) ;
    sum += a [ i ] ;
  }
  return mn * ( sum - mn ) ;
}
